class.jetpack.php

```diff
@@ -2591,6 +2611,11 @@ public static function get_active_modules() {
 		 */
 		$active = apply_filters( 'jetpack_active_modules', $active );
 
+		// Manage is an integral part of how Jetpack works. Can't filter this out.
+		if ( ! in_array( 'manage', $active ) ) {
+			$active[] = 'manage';
```

Comment:
do we need to keep it on the list of modules?

Reply:
I kept this in because @enejb suggested it would make things easier for Calypso. 

His comment here: https://github.com/Automattic/jetpack/pull/7114#issuecomment-299220534

I'd rather remove it, TBH, but it's up to you and enej.

Reply:
if we need it in just for Calypso let's add it WPCOM side?

Reply:
ok! can you do it?

Reply:
:magic: